modules/ztp-gitops-ztp-max-spoke-clusters.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-preparing-the-hub-cluster.adoc]
+
+:_content-type: REFERENCE
+[id="ztp-gitops-ztp-max-spoke-clusters_{context}"]
+= {ztp} managed cluster limits
+
+With {ztp-first}, you can manage thousands of {sno} clusters in geographically displaced regions and networks.
+The scaling limits for the number of clusters that you can manage with a single {rh-rhacm-first} hub cluster will vary depending on various factors affecting the hub cluster.
+For example:
+
+Hub cluster resources::
+Available hub cluster host resources (CPU, memory, storage) are an important factor in determining how many clusters the hub cluster can manage.
+The more resources allocated to the hub cluster, the more managed clusters it can accommodate.
+
+Storage::
+The hub cluster host storage IOPS rating and whether the hub cluster hosts use NVMe storage can affect hub cluster performance and the number of clusters it can manage.
+
+Network bandwidth and latency::
+Slow or high-latency network connections between the hub cluster and managed clusters can impact how the hub cluster manages multiple clusters.
+
+Managed cluster size and complexity::
+The size and complexity of the managed clusters also affects the capacity of the hub cluster.
+Larger managed clusters with more nodes, namespaces, and resources require additional processing and management overhead.
+Similarly, clusters with complex configurations or diverse workloads can require more resources from the hub cluster.
+
+Number of managed policies::
+The number of policies managed by the hub cluster scaled over the number of managed clusters bound to those policies is an important factor that determines how many clusters can be managed.
+
+Monitoring and management workloads::
+{rh-rhacm} continuously monitors and manages the managed clusters.
+The number and complexity of monitoring and management workloads running on the hub cluster can affect its capacity.
+Intensive monitoring or frequent reconciliation operations can require additional resources, potentially limiting the number of manageable clusters.
+
+{rh-rhacm} version and configuration::
+Different versions of {rh-rhacm} can have varying performance characteristics and resource requirements.
+Additionally, the configuration settings of {rh-rhacm}, such as the number of concurrent reconciliations or the frequency of health checks, can impact the managed cluster capacity of the hub cluster.
+
+For details of managed {sno} cluster scale results performed by the Red Hat Performance and Scale lab, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/install/installing#single-node[Creating and managing {sno} clusters with {rh-rhacm}].
```

Comment:
I agree that the linked doc isn't very clear. 

I've reworded this section slightly, and made the managed clusters maximums claim explicit, with a footnote link to the source RHACM docs. @imiller0 WDYT? 

I think this latest version meets the original goal of documenting hub cluster maximums while also adding some extra context to set expectations for customers in real world situations.

Reply:
In my opinion it's pointless to give a date for a specific scale-test but not the _details_ on what was tested. 

Especially if we could provide the details of that scale test, relating to each one of those various considerations you listed and explained, I would feel this would be very valuable.

(If it's not clear, I really like the list of considerations you've produced :+1: )